src/nmodl/main.cpp

```diff
@@ -247,14 +244,6 @@ int main(int argc, const char* argv[]) {
         "Write symbol table to stdout ({})"_format(show_symtab))->ignore_case();
 
     auto codegen_opt = app.add_subcommand("codegen", "Code generation options")->ignore_case();
-    codegen_opt->add_option("--layout",
-        layout,
-        "Memory layout for code generation",
-        true)->ignore_case()->check(CLI::IsMember({"aos", "soa"}));
-    codegen_opt->add_option("--datatype",
-        layout,
-        "Data type for floating point variables",
-        true)->ignore_case()->check(CLI::IsMember({"float", "double"}));
```

Comment:
What do you think of it?

Reply:
do you mean removing data type option as well?

Reply:
Yes, because the output of the option is ... `layout`, copy/paste error, so I guess that no one use it? ^^

Reply:
@alkino : its true but data type will be needed for sure in the near future. So I will keep it for now.